samples/Azure.Management.Storage/readme.md

```diff
@@ -21,4 +21,11 @@ operation-group-to-resource-type:
    Skus: Microsoft.Storage/skus
    Usages: Microsoft.Storage/locations/usages
    PrivateLinkResources: Microsoft.Storage/storageAccounts/privateLinkResources
+operation-group-to-resource:
+   BlobServices: BlobServiceProperties,
```

Comment:
Why is this override needed.  The OperationGroup has a PUT with a body param

https://github.com/Azure/azure-rest-api-specs/blob/e606243e5297312781dd7dbfd7ab76d2329cc088/specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/blob.json#L58

Reply:
It is "BlobService" in swagger while we expect to find its plural.